biz.aQute.bndlib/src/aQute/bnd/osgi/Constants.java

```diff
@@ -307,6 +307,7 @@
 	String							LINK_ATTRIBUTE								= "link";
 	String							LITERAL_ATTRIBUTE							= "literal";
 	String							NAME_ATTRIBUTE								= "name";
+	String							DYNAMIC_ATTRIBUTE							= "dynamic";
```

Comment:
It is not an attribute name. It is a directive value. See https://osgi.org/javadoc/r6/core/org/osgi/framework/namespace/PackageNamespace.html#RESOLUTION_DYNAMIC. So I would prefer to name it `RESOLUTION_DYNAMIC`.